download/tests/support/mod.rs

```diff
@@ -32,25 +34,34 @@ pub fn write_file(path: &Path, contents: &str) {
 }
 
 async fn run_server(addr_tx: Sender<SocketAddr>, addr: SocketAddr, contents: Vec<u8>) {
```

Comment:
IMHO It would be useful if you add a comment explaining this function is adapted from https://hyper.rs/guides/1/server/hello-world. I found it out myself but maybe I wouldn't have to :)

Reply:
Agree. The guide link will be helpful.